tests/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import redis.asyncio as redis
```

Comment:
Ah, nearly through the tests. Seems there's another test run after uninstalling redis, so this might need tweaking to get that working.